thali/thalireplicationmanager.js

```diff
@@ -188,15 +202,62 @@ function syncPeer(peer) {
 
 /* Mux Layer */
 
+function restartMuxServerBridge() {
+  this.stop();
+
+  this.on('stopped', function () {
```

Comment:
Shouldn't this be before line 206? I realize that given the single threaded nature of Javascript it shouldn't theoretically be possible for the event to be emitted before the handler is registered but boy do I hate tempting the gods.

Also shouldn't this be once not on? Otherwise won't this will get called on EVERY stopped event which means that theoretically it isn't possible to ever stop the replication manager?

Oh and what happens if we get a STOP_ERROR? Should we really not try to start again?